<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>randomizer</class>
 <widget class="QDialog" name="randomizer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#2e2e2e;</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
color:#fc7777;
border:2px solid #fc7777;
border-radius:20px;
font: 75 14pt &quot;Courier New&quot;;
}
QPushButton:pressed{
color:#f23333;
}
QPushButton:hover{
background-color:#363636;

}
</string>
   </property>
   <property name="text">
    <string>рандомное число</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#fc7777;
border:2px solid #fc7777;
border-radius:20px;
	font: 14pt &quot;Courier New&quot;;</string>
   </property>
   <property name="maxLength">
    <number>4</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color:#fc7777;

	font: 12pt &quot;Courier New&quot;;</string>
   </property>
   <property name="text">
    <string>начальное число</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#fc7777;
border:2px solid #fc7777;
border-radius:20px;
	font: 14pt &quot;Courier New&quot;;</string>
   </property>
   <property name="maxLength">
    <number>4</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color:#fc7777;

	font: 12pt &quot;Courier New&quot;;</string>
   </property>
   <property name="text">
    <string>начальное число</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color:#fc7777;

	font: 12pt &quot;Courier New&quot;;</string>
   </property>
   <property name="text">
    <string>рандомное число</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_out">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#fc7777;
border:2px solid #fc7777;
border-radius:20px;
	font: 14pt &quot;Courier New&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>2</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#fc7777;
border:2px solid #fc7777;
border-radius:20px;
font: 75 14pt &quot;Courier New&quot;;</string>
   </property>
   <property name="text">
    <string>рандомное число </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
